test/testUnregister.js

```diff
@@ -46,4 +50,29 @@ describe('mercurius unregister', function() {
     })
     .expect(404, done);
   });
+
+  it('deletes all token related data after a registration has been removed', function(done) {
+    redis.exists(token)
+    .then(function(exists) {
+      exists.should.equal(0);
+      console.log('ZZZ: checking ' + token);
+      return redis.exists(token + ':clients');
+    })
+    .catch(done)
```

Comment:
You can remove all `catch`es and `done`s and simply return the promise. `Mocha` automatically handles promise resolution as a pass and promise rejection as a failure.